server/main.py

```diff
@@ -32,36 +35,72 @@ class Payload(BaseModel):
     trot: Optional[float] = None
     path_length: float
     simulate_slit: Optional[int] = None
-    mode: Literal["absorbance", "transmittance_noslit", "radiance_noslit", "transmittance", "radiance"]
+    use_simulate_slit: bool
+    mode: Literal[
+        "absorbance",
+        "transmittance_noslit",
+        "radiance_noslit",
+        "transmittance",
+        "radiance",
+    ]
     database: Literal["hitran", "geisa"]
-    use_simulate_slit: bool = False
 
 
+# calculating the spectrum return back the spectrum
+def calculate_spectrum(payload):
+    print(">> Payload : ")
+    print(payload)
+    spectrum = radis.calc_spectrum(
+        payload.min_wavenumber_range,
+        payload.max_wavenumber_range,
+        molecule=[species.molecule for species in payload.species],
+        mole_fraction={
+            species.molecule: species.mole_fraction for species in payload.species
+        },
+        # TODO: Hard-coding "1,2,3" as the isotopologue for the time-being
+        isotope={species.molecule: "1,2,3" for species in payload.species},
+        pressure=payload.pressure,
+        Tgas=payload.tgas,
+        Tvib=payload.tvib,
+        Trot=payload.trot,
+        path_length=payload.path_length,
+        export_lines=False,
+        wstep="auto",
+        databank=payload.database,
+        use_cached=True,
+    )
+    return spectrum
+
+
+# create the folder in server for better organization
+def create_download_directory():
+    if os.path.exists("DOWNLOADED_SPECFILES"):
```

Comment:
Since we repeat this magic string three times, it would be a good candidate to set as a module-level variable like `DOWNLOADED_SPECFILES_DIRECTORY = "DOWNLOADED_SPECFILES"`.